README.md

```diff
@@ -51,6 +51,27 @@ If you want to work on improving the components it’s best to run the latest co
     - In the repository of an app do `npm link @nextcloud/vue` (you need to re-link any time you do `npm ci` in the app)
 4. Then build the app with: `npm run build` (or watch for changes with `npm run watch`)
 
+### Using vue-devtools
+
+If you want to use [vue-devtools](https://github.com/vuejs/vue-devtools) in your browser, you need to patch your nextcloud instance as follow:
```

Comment:
This is only for firefox

Reply:
Oh yes, they said something about that in the chat.

Is there something else to do for Chrome,... or nothing needed for those browsers?

Reply:
no, works for chrome :)

Reply:
ok, PR updated